src/Windowing/Silk.NET.Windowing.Common/Internals/ViewImplementationBase.cs

```diff
@@ -213,6 +213,52 @@ public bool VSync
             }
         }
 
+        public IGLContext? GLContext
+        {
+            get
+            {
+                if (!IsInitialized && (API.API == ContextAPI.OpenGL || API.API == ContextAPI.OpenGLES))
+                {
+                    static void Throw() => throw new InvalidOperationException
+                    (
+                        "OpenGL functions can only be used after initialization (just before the Load callback is " +
+                        "executed)"
+                    );
+
+                    Throw();
+                }
+                else if (API.API != ContextAPI.OpenGL && API.API != ContextAPI.OpenGLES)
+                {
+                    return null;
+                }
```

Comment:
I think throwing even if Vk is requested makes sense. It removes some checks here, and makes more sense from the callers side.

Reply:
Fine with me.